compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/AnnotationDesugar.java

```diff
@@ -107,6 +102,14 @@ protected void rewritePackageAnnotations(BLangPackage pkgNode) {
             initFunction.body.stmts.add(index++, stmt);
         }
 
+        // Handle service annotations
```

Comment:
here the order matters right?, better to put a comment specifying that, and the order